manifests/a/aardappel/lobster/2024.0/aardappel.lobster.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: aardappel.lobster

```

```suggestion
PackageIdentifier: Aardappel.lobster
```
Please update file name and path to match this change.